app.js

```diff
@@ -36,6 +47,34 @@ app.set('engine', engine.engine);
 app.get('/health', routeGetHealth);
 app.post('/contract', routePostContract);
 app.post('/token', routePostToken);
+// TODO: do something better than 'bind' to pass the manager to the route
```

Comment:
Are we going to take this out before merging to `master`? I think it would be better if code that is meant to be removed is simply not added in the first place.